tests/integration/ovs_test.py

```diff
@@ -144,6 +144,22 @@ def test_vlan_as_ovs_bridge_slave(vlan_on_eth1):
         assertlib.assert_state_match(state)
 
 
+@pytest.mark.xfail(
+    raises=(NmstateLibnmError, AssertionError),
+    reason="https://bugzilla.redhat.com/1724901",
+)
+def test_ovs_interface_with_max_length_name():
+    bridge = Bridge(BRIDGE1)
+    ovs_interface_name = "ovs123456789012"
```

Comment:
Have you also tried with more than 15 chars?

Reply:
No, that should fail on the internal interface.

Reply:
Nvm, thank you for the explanation :-)